admob/src/include/firebase/admob/types.h

```diff
@@ -186,6 +190,10 @@ class AdResult {
   /// Gets the message describing the error.
   const std::string& message() const;
 
+  /// Gets the ResponseInfo if an loadAd error occurred, with a collection of
```

Comment:
Will there always be a responseinfo? What if it's the situation where it wasn't, will it just be empty?

Reply:
Yes, the call to this method will return an emtpy ResponseInfo object.

Reply:
Updated the comments.